src/walletSdk/Customer/index.ts

```diff
@@ -26,14 +31,36 @@ export class Sep12 {
     };
   }
 
-  async getByIdAndType(id: string, type: string): Promise<GetCustomerResponse> {
+  async getCustomer({
+    id,
+    type,
+    memo,
+    lang,
+  }: GetCustomerParams): Promise<GetCustomerResponse> {
+    if (!id && !type && !memo) {
```

Comment:
@acharb @Ifropc  I'm not sure we should be enforcing any of those params, I could see Anchors being able to detect clients by its Sep-10 auth token. What do you think?

Reply:
ya I agree on second thought, removed:

https://github.com/stellar/typescript-wallet-sdk/pull/71/commits/f58a1895a258b6c115e1beec6601f30e80f284e4

Reply:
True